.docker/php/80-fpm-dev/Dockerfile

```diff
@@ -1,10 +1,12 @@
-FROM docker.pkg.github.com/fogospt/fogosapi/php:8.0-fpm-prod
+FROM ghcr.io/fogospt/fogosapi-php:8.0-fpm-prod
```

Comment:
Can you help me understand the differences between these two and why using ghcr.io might be better? I wasn't aware of the existent of that service from GitHub!

Reply:
The ghcr.io is the new container registry from github and it has a different set of features. 
In summary:
```
Comparing the Docker registry and the Container registry:

The Container registry (https://ghcr.io) offers improved performance for containers and will supercede the Docker registry in the future.
You can migrate your Docker images and workflows from the Docker registry (docker.pkg.github.com) to the Container registry (https://ghcr.io) for granular permissions, container storage optimization, and more. For more information, see "Migrating to the Container registry from the Docker registry."
```
For more info you can check these docs:

- [migrating-to-the-container-registry-from-the-docker-registry]( https://docs.github.com/en/packages/working-with-a-github-packages-registry/migrating-to-the-container-registry-from-the-docker-registry#key-differences-between-the-container-registry-and-the-docker-registry)
- [Container registry
](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry)